Topics/Tech_Stacks.md

```diff
@@ -1,3 +1,52 @@
 ## Tech Stacks
 
 ### [Learning MySQL for databases](./Tech_Stacks/Learning_MySQL.md)
+
+## Tech Stacks
+
+### CSS (Cascading Style Sheets)
+
+CSS, or cascding style sheets, are used to add styles, like spacing and layout, to HTML web pages. While CSS can be done through in-line styles, it is common convention to have a style sheet for the webpage, or the style tag. 
+
+CSS rules consist of a selector and a declaration block, where the selector points to the HTML element or set of elements you want to style and the declaration denotes the style. 
+
+<img width="444" alt="Capture" src="https://user-images.githubusercontent.com/81656509/226232503-8d496905-038e-4f61-b17f-d4f702f75ecf.PNG">
+
+
+Image source: https://www.w3schools.com/css/css_syntax.asp 
+
+The link below provides interactive examples and exercises for basic CSS.
+
+W3Schools link: https://www.w3schools.com/css/ 
+
+#### CSS Cascade
+
+Because of the existence of ids and classes in selectors, often multiple CSS declaratiosn can apply to an HTML element, and they can conflict with each other. The browser needs to resolve these conflicts in order to display the final style onto the website, and this is done through CSS Cascade, which determines which declarations take precedence over others 
```

Comment:
```suggestion
Because of the existence of ids and classes in selectors, often multiple CSS declarations can apply to an HTML element, and they can conflict with each other. The browser needs to resolve these conflicts in order to display the final style onto the website, and this is done through CSS Cascade, which determines which declarations take precedence over others.
```